time stamp
uint8 index
uint8 duration
uint32 code
int16 error_x
int16 error_y
int16 error_yaw
uint8 sum